main/res/values/strings.xml

```diff
@@ -1423,8 +1423,8 @@
 
     <string name="cache_whereyougo_start">Open WhereYouGo to download and start this cartridge</string>
     <string name="cache_whereyougo_install">Install WhereYouGo player to download and start this cartridge</string>
-    <string name="cache_chirpwolf_start">Open Chirp Wolf to read the chirp data for this cache</string>
-    <string name="cache_chirpwolf_install">Install Chirp Wolf to be able to read chirp data</string>
+    <string name="cache_chirpwolf_start">This cache might use a chirp transmitter. Open Chirp Wolf to read the chirp data for this cache</string>
```

Comment:
New string is ok for me, IMHO.

If we want to stress possible alternatives a bit more, we could use something like: 

"_This cache might use a chirp transmitter or other wireless beacons._ Open Chirp Wolf to read the chirp data for this cache _(if available)_." and "_This cache might use a chirp transmitter or other wireless beacons._ Install Chirp Wolf to be able to read chirp data."

(Highlighting changes for visibility only, not to be included in the actual program.)

Reply:
I first was afraid to make it to long. 
But as this is only shown for those specific caches, I guess its fine. Will update the PR.